crates/core/Cargo.toml

```diff
@@ -11,10 +11,9 @@ crate-type = ["cdylib"]
 
 [dependencies]
 anyhow = { workspace = true }
-javy = { workspace = true, features = ["export_alloc_fns", "json"] }
-once_cell = { workspace = true }
+javy-plugin-api = { workspace = true, features = ["json"] }
```

Comment:
Is it intended to have this dependency at the workspace level? I think it's not shared across any other crates in the workspace, right?

Reply:
I can move it out of the workspace into `javy-core`. For some reason I thought the `javy` crate was only used by `javy-core` but I can see after looking that the fuzzer also uses it.